web/src/cypress/support/index.ts

```diff
@@ -35,12 +31,18 @@ import './page-search'
 import './page-action'
 import './page-nav'
 import './page-fab'
-import './config'
 import './sql'
 import './form'
 import './dialog'
-import './limits'
 
+export * from './config'
+export * from './limits'
 export * from './util'
 
 import './fail-fast'
+Cypress.env('RETRIES', 2)
+Cypress.Cookies.defaults({
+  whitelist: 'goalert_session.2',
+})
+
+export * from './util'
```

Comment:
duplicate of [line 40](https://github.com/target/goalert/pull/520/files#diff-e433f8c67e8c459348b83b0482bee448R40)?